cmd/traffic_cache_tool/CacheDefs.cc

```diff
@@ -229,15 +229,15 @@ Stripe::InitializeMeta()
 {
   Errata zret;
   // memset(this->raw_dir, 0, dir_len);
-  for (int i = 0; i < 2; i++) {
-    for (int j = 0; j < 2; j++) {
-      _meta[i][j].magic             = StripeMeta::MAGIC;
-      _meta[i][j].version.ink_major = ts::CACHE_DB_MAJOR_VERSION;
-      _meta[i][j].version.ink_minor = ts::CACHE_DB_MINOR_VERSION;
-      _meta[i][j].agg_pos = _meta[i][j].last_write_pos = _meta[i][j].write_pos = this->_content;
-      _meta[i][j].phase = _meta[i][j].cycle = _meta[i][j].sync_serial = _meta[i][j].write_serial = _meta[i][j].dirty = 0;
-      _meta[i][j].create_time = time(nullptr);
-      _meta[i][j].sector_size = DEFAULT_HW_SECTOR_SIZE;
+  for (auto &i : _meta) {
```

Comment:
I was hoping @persiaAziz would do something like
```
for (auto i : {A, B}) {
  for (auto j : { HEAD, FOOT}) {
    _meta[i][j] = StripeMeta::MAGIC;
// ....
```
I suppose even with that you'd want to create `auto & item = _meta[i][j];` rather than indexing repeatedly.